### ISSUE TITLE
The code of loss function

### ISSUE THREAD

**Issue by CHNxindong** [YanzuoLu/CFLD#15]
Thanks for your great work and released code! 

I have two problems for code in pose_transfer_train.py: 
1. the losses said in paper are reconstruction loss and mse loss:
![image](https://github.com/YanzuoLu/CFLD/assets/20262070/8005d38e-ad51-4734-8a4d-005c3fa2dafe)
![image](https://github.com/YanzuoLu/CFLD/assets/20262070/8841cf16-4a5b-43dc-9c94-5c61f6c5faac)
![image](https://github.com/YanzuoLu/CFLD/assets/20262070/eaacf897-de3a-4e19-a24f-0453c9c92b09)
But there is only 1 line code in implementation:
![image](https://github.com/YanzuoLu/CFLD/assets/20262070/f0e2611b-3a92-485a-8a98-73ab998de569)

2. Why do the pose_img_src and pose_img_tgt are concated for pose encoder? And why do the img_src and img_tgt are concated for input? 
![image](https://github.com/YanzuoLu/CFLD/assets/20262070/4a6827bc-48f4-4e96-8d27-a25f659066dc)
![image](https://github.com/YanzuoLu/CFLD/assets/20262070/10c14a85-34f4-4721-b9a8-795bf2a65cf5)

**Comment by ButoneDream** [comment]
我也想知道

**Comment by YanzuoLu** [comment]
I think your second question already corresponds to the answer to your first question.
It can be seen that the only difference between eq.(3) and (5) is the target pose.
These two target poses correspond to the original image and the target image in the training pair respectively.
**In the diffusion model, the latents sent to unet during training are the noisy generated target images, which is the noisy form of the groundtruth image.**
Hope this helps.

**Comment by CHNxindong** [comment]
I see it. Thanks for your quick and patient reply.

**Comment by colorful-liyu** [comment]
Hello, sorry to bother you.
I have tried contacting the DeepFashion project managers to request the unzip password, but unfortunately I haven’t received any response. May I kindly ask if you would be able to share the password with me?

I am a PhD student conducting research in the field of fashion and clothing, and this dataset would be very important for my work. Thank you very much in advance for your help!

My email: melodious@sjtu.edu.cn
